### PR TITLE
📝 Add setup instructions

### PR DESCRIPTION
**Pull request by afranke**
Signed-off-by: Alexandre Franke <alexandre.franke@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,18 @@ gets one, downloads an artifact and unpacks it.
 
 Based on [a Buildkite version](https://github.com/matrix-org/buildkite-webhook-listener).
 
+## Github setup
+
+Go to the project “Settings”, in the “Webhooks” section, and click on the “Add webhook” button.
+Make sure you set the “Content type” to `application/x-www-form-urlencoded`.
```
Comment (richvdh) [inline README.md:11]: json, no?

File: README.md [hunk 2]
```diff
@@ -5,6 +5,18 @@ gets one, downloads an artifact and unpacks it.
...
+The “Secret” is what you’ll pass as `--webhook-token` to the listener.
+Choose the “Let me select individual events” option and check *only* the “Workflow runs” box.
+
+
+## Command-line options
+
+The API token passed on the command line is a [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) with `repo` access, from a Github user that can read the repository.
```
Comment (richvdh) [inline README.md:18]:
```suggestion
The API token passed as `--api-token` on the command line should be a 
[Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) 
with `repo` access, from a Github user that can read the repository.
```

File: README.md (hunk 1 above)
Comment (afranke) [reply README.md:11]: :man_facepalming:

**Review by richvdh — APPROVED** [review]
fab, thanks!

**Pull request merged**